vimeo/VimeoUpload
# VIM-3501: Download Mayhem

### Ticket

See [this companion PR](https://github.vimeows.com/MobileApps/Vimeo-iOS/pull/518)
### Ticket Summary

See [this companion PR](https://github.vimeows.com/MobileApps/Vimeo-iOS/pull/518)
### Implementation Summary

See [this companion PR](https://github.vimeows.com/MobileApps/Vimeo-iOS/pull/518)

This PR provides library work necessary for the above linked companion PR. 
### How to Test

See [this companion PR](https://github.vimeows.com/MobileApps/Vimeo-iOS/pull/518)
